.gitignore

```diff
@@ -11,3 +11,4 @@ __pycache__
 *.iml
 .project
 .vagrant
+/Anton_Negovora/10.K8S.WS
```

Comment:
Please remove this line, it is common file, your change will migrate to another repositories